src/views/Graph.vue

```diff
@@ -89,17 +89,19 @@ const config = {
     {
       selector: 'node',
       css: {
-        'background-image': function(node) {
+        'background-image': function (node) {
           let path = node.data('icon')
           // TODO: not sure if we can provide the path as a normal string, without importing the svg here?
-          path === undefined || path === '' ? path = require('@/../public/img/baseline-donut_large-24px.svg') : ''
+          if (path === undefined || path === '') {
+            path = require('@/../public/img/baseline-donut_large-24px.svg')
+          }
           // console.log('ICON PATH --> ', path)
           return path
         },
         'background-fit': 'contain contain',
-        'background-image-opacity': function(node) {
+        'background-image-opacity': function (node) {
           let opacity
-          node.data('running') > 0 ? opacity =  1 : opacity = .6
+          node.data('running') > 0 ? opacity = 1 : opacity = 0.6
```

Comment:
Can we just do this?

```suggestion
          return node.data('running') > 0 ? 1.0 : 0.6
```

Reply:
Oh, I think we can adjust this one too :+1: